rapidpro_community_portal/static/css/main.css

```diff
@@ -184,6 +193,7 @@ ol{
 .rich-text p{
   padding-top:5px;
   padding-bottom:5px;
+  font-size:16px;
```

Comment:
Is there a reason we can't do em here? Seems like we use em everywhere else and it is preferred CSS.